src/quickunlock/Polkit.cpp

```diff
@@ -149,6 +173,7 @@ bool Polkit::getKey(const QString& dbPath, QByteArray& key)
         Botan::secure_scrub_memory(keychainKey.data(), keychainKey.size());
         Botan::secure_scrub_memory(keychainIv.data(), keychainIv.size());
         Botan::secure_scrub_memory(keychainBytes.data(), keychainBytes.size());
+        Botan::secure_scrub_memory(encryptedMasterKey.data(), encryptedMasterKey.size());
```

Comment:
Hmm.. should these also be scrubbed when removed from `m_encryptedMasterKeys`? @droidmonkey , what's your opinion - is this overkill?

Reply:
Anything encrypted in memory is already "random" data so it doesn't matter.